src/modules/address-book/components/AddressBookAddEdit.vue

```diff
@@ -143,34 +144,45 @@ export default {
     nicknameRules() {
       return [
         value =>
-          value.length < 20 ||
+          (value && value.length < 20) ||
           this.$t('interface.address-book.validations.nickname-length')
       ];
     },
     validAddress() {
       return this.resolvedAddr.length > 0
-        ? utils.isAddress(this.resolvedAddr)
-        : utils.isAddress(this.addressToAdd);
+        ? utils.isAddress(this.resolvedAddr) &&
+            utils.isHexStrict(this.resolvedAddr)
+        : utils.isAddress(this.addressToAdd) &&
+            utils.isHexStrict(this.addressToAdd);
```

Comment:
why are we checking for `isHexStrict`?

Reply:
because of this bug https://app.asana.com/0/1163629870679883/1200627642947378 hahah should we just allow them to add those addresses?

Reply:
yea but isAddress should cover it

Reply:
hmmm when I tried isAddress on there , it returned true but i can try again

Reply:
its returning true for 1254155464511115555555555555555555555511 D:

Reply:
weird, k let me take a look

Reply:
> its returning true for 1254155464511115555555555555555555555511 D:

yea you are right, https://web3js.readthedocs.io/en/v1.4.0/web3-utils.html#id32

I changed it to our custom isAddress, which check for isHexStrict. Can you add a debounce to name resolving right now it tries to resolve on every keypress after `.`

Reply:
yep! after playing around with it , i think i need to change something from components first

Reply:
helloo I pushed something new and updated components but I can't update the package here yet until the overlay changes are approved , will update after!